backend/config.go

```diff
@@ -52,16 +55,22 @@ func (r *Replica) NewView(msg consensus.SyncInfo) {
 		return
 	}
 	var ctx context.Context
-	r.newviewCancel()
-	ctx, r.newviewCancel = context.WithCancel(context.Background())
+	r.newViewCancel()
+	ctx, r.newViewCancel = context.WithCancel(context.Background())
 	r.node.NewView(ctx, hotstuffpb.SyncInfoToProto(msg), gorums.WithNoSendWaiting())
 }
 
+// Metadata returns the gRPC metadata from this replica's connection.
+func (r *Replica) Metadata() map[string]string {
+	return r.md
+}
+
 // Config holds information about the current configuration of replicas that participate in the protocol,
 // and some information about the local replica. It also provides methods to send messages to the other replicas.
 type Config struct {
-	mods    *consensus.Modules
-	optsPtr *[]gorums.ManagerOption // using a pointer so that options can be GCed after initialization
+	mods      *consensus.Modules
+	opts      []gorums.ManagerOption // using a pointer so that options can be GCed after initialization
```

Comment:
This no longer uses a pointer to the slice; maybe the comment should be removed or rephrased?